packages/web/src/components/cell/cellTypes.ts

```diff
@@ -52,11 +52,11 @@ export type CellProps<
 >
 
 export type CellLoadingProps<TVariables extends OperationVariables = any> = {
-  queryResult?: NonSuspenseCellQueryResult<TVariables> | SuspenseCellQueryResult
+  queryResult?: NonSuspenseCellQueryResult<any, TVariables> | SuspenseCellQueryResult
```

Comment:
I think we probably aught to have a type parameter on `CellLoadingProps` for `TData` here as well. And ideally it should come first on `CellLoadingProps`, but that'd be a breaking change 😒